web/app/js/components/ResourceDetail.jsx

```diff
@@ -420,7 +420,7 @@ export class ResourceDetailBase extends React.Component {
         <MetricsTable
           resource="pod"
           title="TCP"
-          isTcpTable={true}
+          isTcpTable
```

Comment:
Will this produce proper XHML?

Reply:
Yep, this was caught by the [react/jsx-boolean-value](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) rule. When setting a boolean prop to `true` in react, there's no need to explicitly pass `true` as the prop value, so this rule removes it.